cypress_shared/pages/page.ts

```diff
@@ -212,4 +212,14 @@ export default abstract class Page {
         })
     })
   }
+
+  clearDateInputs(prefix: string): void {
```

Comment:
Noice!

Reply:
copied from TA. 
Didn't realise this was neccessary until @patrickjfl made me aware Cypress doesn't do it automatically